build-tools/installers/create-installers.targets

```
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\Configuration.props" />
+  <Import Project="..\scripts\XAVersionInfo.targets" />
+  <Import Project="..\..\src\mono-runtimes\ProfileAssemblies.projitems" />
+  <UsingTask AssemblyFile="..\..\bin\Build$(Configuration)\xa-prep-tasks.dll"   TaskName="Xamarin.Android.BuildTools.PrepTasks.ReplaceFileContents" />
+  <PropertyGroup>
+    <RootBuildDir>$(XamarinAndroidSourcePath)\bin\$(Configuration)\</RootBuildDir>
+    <FrameworkSrcDir>$(RootBuildDir)\lib\xamarin.android\xbuild-frameworks\MonoAndroid</FrameworkSrcDir>
+    <MSBuildSrcDir>$(RootBuildDir)\lib\xamarin.android\xbuild\Xamarin\Android</MSBuildSrcDir>
+    <FirstInstallerFrameworkVersion Condition="Exists('$(FrameworkSrcDir)\$(AndroidFirstFrameworkVersion)')">$(AndroidFirstFrameworkVersion)</FirstInstallerFrameworkVersion>
+    <FirstInstallerFrameworkVersion Condition="'$(FirstInstallerFrameworkVersion)' == ''">$(AndroidLatestStableFrameworkVersion)</FirstInstallerFrameworkVersion>
+    <BclFrameworkVersion>v1.0</BclFrameworkVersion>
+    <LibExtension Condition=" '$(HostOS)' == 'Darwin' ">dylib</LibExtension>
+    <LibExtension Condition=" '$(HostOS)' == 'Linux' ">so</LibExtension>
+    <LibExtension Condition=" '$(HostOS)' == 'Windows' ">dll</LibExtension>
+  </PropertyGroup>
+  <ItemGroup>
+    <FrameworkFiles Include="$(FrameworkSrcDir)\*\AndroidApiInfo.xml"></FrameworkFiles>
+    <FrameworkFiles Include="$(FrameworkSrcDir)\*\Mono.Android.*"></FrameworkFiles>
+    <FrameworkFiles Include="$(FrameworkSrcDir)\**\FrameworkList.xml"></FrameworkFiles>
+    <FirstFrameworkFiles Include="$(FrameworkSrcDir)\$(FirstInstallerFrameworkVersion)\Mono.Android.Export.dll"></FirstFrameworkFiles>
+    <FirstFrameworkFiles Include="$(FrameworkSrcDir)\$(FirstInstallerFrameworkVersion)\OpenTK*.dll"></FirstFrameworkFiles>
+    <BCLFacadeFiles Include="@(MonoFacadeAssembly->'$(FrameworkSrcDir)\$(BclFrameworkVersion)\Facades\%(Identity)')"></BCLFacadeFiles>
+    <BCLFrameworkFiles Include="@(MonoProfileAssembly->'$(FrameworkSrcDir)\$(BclFrameworkVersion)\%(Identity)')"></BCLFrameworkFiles>
+    <BCLFrameworkFiles Include="$(FrameworkSrcDir)\$(BclFrameworkVersion)\Java.Interop.dll"></BCLFrameworkFiles>
+    <BCLFrameworkFiles Include="$(FrameworkSrcDir)\$(BclFrameworkVersion)\Mono.Data.Sqlite.dll.config"></BCLFrameworkFiles>
+    <BCLFrameworkFiles Include="$(FrameworkSrcDir)\$(BclFrameworkVersion)\Mono.Posix.dll"></BCLFrameworkFiles>
+    <BCLFrameworkFiles Include="$(FrameworkSrcDir)\$(BclFrameworkVersion)\System.EnterpriseServices.dll"></BCLFrameworkFiles>
+    <BCLFrameworkFiles Include="$(FrameworkSrcDir)\$(BclFrameworkVersion)\Xamarin.Android.NUnitLite.dll"></BCLFrameworkFiles>
+  </ItemGroup>
+  <ItemGroup>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\android-support-multidex.jar"></MSBuildFiles>
```

Coding style: Items only need an explicit closing tag if item metadata is specified.

No item metadata is specified here (and elsewhere).  As such, `/>` should be used:

```suggestion
    <MSBuildFiles Include="$(MSBuildSrcDir)\android-support-multidex.jar" />
```

---

Coding style changes have been applied.